.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
     rev: v2.4.0
     hooks:
       - id: prettier
-        args: [--write, ./frontend]
+        args: ["--ignore-path ./frontend/.prettierignore --write ./frontend"]
```

Comment:
Pass all args as one string to avoid bugs. Previously we were linting the whole frontend repo every time, which is why every pre-commit was slow. This change also passes `.prettierignore` file